### PR TITLE
rvic-daccs 1.1.4

### LINKED ISSUE #65: Increase RVIC version to 1.1.4
rvic-daccs 1.1.4 is released. Target the version number in requirements.txt and adapt the changes in osprey processes.

### PR DESCRIPTION
**Pull request by sum1lim**
resolves #65 

`osprey_i65` is running in port 30124
The fixes in jupyter lab demo are irrelevant to the issue but included here since they are minor issues.

### CODE REVIEW

**Review by cairosanders — APPROVED** [review]
Works for me! :tada:

File: osprey/processes/wps_convert.py [hunk 1]
```diff
@@ -122,10 +122,7 @@ def _handler(self, request, response):
             process_step="process",
         )
 
-        try:
-            convert(config_file)
-        except RecursionError:
-            close_logger()
+        convert(config_file)
```
Comment (nikola-rados) [inline osprey/processes/wps_convert.py:125]: Much better having this handled in the actual package. You can remove the import for `close_logger` if we are no longer using it.

File: osprey/processes/wps_convolution.py [hunk 2]
```diff
@@ -148,10 +148,7 @@ def _handler(self, request, response):
             process_step="process",
         )
 
-        try:
-            convolution(config)
-        except RecursionError:
-            close_logger()
+        convolution(config)
```
Comment (nikola-rados) [inline osprey/processes/wps_convolution.py:151]: Same comment as above!

File: osprey/processes/wps_full_rvic.py [hunk 3]
```diff
@@ -232,10 +232,8 @@ def _handler(self, request, response):
             log_level=loglevel,
             process_step="parameters_process",
         )
-        try:
-            parameters(params_config, np)
-        except RecursionError:
-            close_logger()
+
+        parameters(params_config, np)
```
Comment (nikola-rados) [inline osprey/processes/wps_full_rvic.py:236]: Same as above, you can remove the import.

File: osprey/processes/wps_parameters.py [hunk 4]
```diff
@@ -184,10 +184,8 @@ def _handler(self, request, response):
             log_level=loglevel,
             process_step="process",
         )
-        try:
-            parameters(config, np)
-        except RecursionError:
-            close_logger()
+
+        parameters(config, np)
```
Comment (nikola-rados) [inline osprey/processes/wps_parameters.py:188]: Same as above.

File: requirements.txt [hunk 5]
```diff
@@ -2,6 +2,6 @@ pywps>=4.2
 jinja2
 click
 psutil
-rvic-daccs==1.1.2
+rvic-daccs==1.1.4
```
Comment (nikola-rados) [inline requirements.txt:5]: :+1:

**Review by nikola-rados — APPROVED** [review]
This is ready to go, you just may want to remove the imports that are no longer used. Nice work!

**Pull request merged**